docarray/array/storage/base/backend.py

```diff
@@ -41,6 +42,7 @@ def _init_subindices(
                     config, config_subindex, config_joined, name
                 )
                 self._subindices[name] = self.__class__(config=config_joined)
+                self._subindices[name]._is_subindex = True
```

Comment:
this should be handled somehow in the constructor

Reply:
this shoul dbe another parameter in `__init_storage` with private name `_is_subindex` with `False` as default